openedx/adg/admin_dashboard/helpers.py

```diff
@@ -25,3 +25,20 @@ def get_http_response_with_csv_file_attachment(file_data, file_name):
     response = HttpResponse(buf.getvalue(), content_type='text/csv')
     response['Content-Disposition'] = f'attachment; filename={file_name}'
     return response
+
+
+def add_query_param_for_target_page(params, page_number):
+    """
+    Returns an encoded url with all the original query params and adds the target page number as a query
+    param as well
+
+    Arguments:
+        params (QueryDict): Contains all the query params
+        page_number: Target page number to show in the param
```

Comment:
probably an int, but still it'll be good if we can add the datatype here.

Reply:
Missed it, adding right now